src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff.Tool/Microsoft.DotNet.ApiDiff.Tool.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetToolMinimum)</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <!-- <IsPackable>true</IsPackable>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>genapidiff</ToolCommandName> -->
+    <PackageDescription>Tool to emit markdown diffs between sets of assemblies.</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\GenAPI\Microsoft.DotNet.GenAPI\Microsoft.DotNet.GenAPI.csproj" />
+    <ProjectReference Include="..\..\Microsoft.DotNet.ApiSymbolExtensions\Microsoft.DotNet.ApiSymbolExtensions.csproj" />
+    <ProjectReference Include="..\Microsoft.DotNet.ApiDiff\Microsoft.DotNet.ApiDiff.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.CommandLine" VersionOverride="2.0.0-beta4.22272.1" />
```

Comment:
Why the version override?

Reply:
This is the latest version, it's the one documented in the official docs, and it has a completely different API surface (they introduce public API breaking changes).

We should consider updating our other Compat tools to using this version. I opened an issue to track this: https://github.com/dotnet/sdk/issues/46482

Reply:
Why not use the same one as the rest of the repo, apicompat and genapi? https://github.com/dotnet/sdk/blob/4df298b776b2f691b5b92e334104d584c722ef43/src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompat.Tool/Microsoft.DotNet.ApiCompat.Tool.csproj#L19 & https://github.com/dotnet/sdk/blob/4df298b776b2f691b5b92e334104d584c722ef43/src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI.Tool/Microsoft.DotNet.GenAPI.Tool.csproj#L13

Reply:
It's too old. I think we should be testing latest whenever possible. I don't think it hurts to override the version. Eventually all other tools will have to use latest, ideally.

Reply:
Updated to consume the latest updated last week. I had to also update the usage because the API surface changed considerably.